migrations/20240405110006_vnft_into_ud_tbl.sql

```diff
@@ -0,0 +1,128 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+
+SET search_path = devices_api, public;
+CREATE TABLE vehicle_nfts_backup AS SELECT *  FROM vehicle_nfts;
+
+ALTER TABLE user_devices
+    ADD COLUMN mint_request_id char(27) CONSTRAINT user_devices_mint_request_id_fkey UNIQUE REFERENCES meta_transaction_requests(id);
+
+ALTER TABLE user_devices
+    ADD COLUMN burn_request_id char(27) CONSTRAINT user_devices_burn_request_id_fkey UNIQUE REFERENCES meta_transaction_requests(id);
+
+ALTER TABLE user_devices
+    ADD COLUMN token_id numeric(78, 0) CONSTRAINT user_devices_token_id_key UNIQUE;
+
+ALTER TABLE user_devices
+    ADD COLUMN claim_id varchar CONSTRAINT user_devices_claim_id_fkey UNIQUE REFERENCES verifiable_credentials(claim_id);
+
+ALTER TABLE user_devices
+    ADD COLUMN owner_address BYTEA CONSTRAINT user_devices_owner_address_check CHECK (length(owner_address) = 20);
+
+UPDATE user_devices 
+    SET 
+        burn_request_id = vnft.burn_request_id,
+        mint_request_id = vnft.mint_request_id,
+        token_id = vnft.token_id,
+        claim_id = vnft.claim_id,
+        owner_address = vnft.owner_address
+FROM vehicle_nfts vnft 
+WHERE vnft.user_device_id = user_devices.id;
+
+ALTER TABLE aftermarket_devices DROP CONSTRAINT autopi_units_vehicle_token_id_fkey;
+ALTER TABLE aftermarket_devices ADD CONSTRAINT autopi_units_vehicle_token_id_fkey FOREIGN KEY (vehicle_token_id) REFERENCES user_devices(token_id) ON DELETE CASCADE;
+
+ALTER TABLE synthetic_devices DROP CONSTRAINT fkey_vehicle_token_id;
+ALTER TABLE synthetic_devices ADD CONSTRAINT fkey_vehicle_token_id FOREIGN KEY (vehicle_token_id) REFERENCES user_devices(token_id);
+
+DROP TABLE vehicle_nfts;
+DROP TABLE vehicle_nfts_backup;
+
+CREATE MATERIALIZED VIEW vehicle_nfts
```

Comment:
Would a (non-materialized) view be much worse for our use case?

Reply:
I was thinking it would be faster to have a materialized view but I think you're right-- since we would have to refresh it all the time anyways, a regular view probably makes more sense

Reply:
My unscientific argument is that it's kinda just `user_devices` with some columns removed.